tests/model_fields_/test_embedded_model.py

```
@@ -82,18 +92,10 @@ def test_pre_save(self):
         self.assertEqual(obj.simple.auto_now_add, auto_now_add)
         self.assertGreater(obj.simple.auto_now, auto_now_two)
 
-    def test_foreign_key_in_embedded_object(self):
```

Why the deleted test?

---

Inside of embedded object we cannot have foreign keys, If we want to support it, I think I have to add some validations in the try_transform. Maybe we have to raise an exception ?

---

Let's raise an exception for now if a foreignkey is placed inside an embedded object. Otherwise, devs may try and it will fail.

---

The new test expects an exception. 😄